node/forceios.js

```diff
@@ -118,8 +118,8 @@ function createHybridApp(config) {
 
     shelljs.exec('cordova create "' + projectDir + '" ' + config.companyid + ' ' + config.appname);
     shelljs.pushd(projectDir);
-    shelljs.exec('cordova platform add ios');
-    shelljs.exec('cordova plugin add https://github.com/forcedotcom/SalesforceMobileSDK-CordovaPlugin');
+    shelljs.exec('cordova platform add ios@' + cordovaPlatformVersion);
+    shelljs.exec('cordova plugin add https://github.com/forcedotcom/SalesforceMobileSDK-CordovaPlugin#unstable');
```

Comment:
Developers might have different versions of CLI installed. Installing CLI across different platforms is too painful to support. Besides, I wouldn't go that far to step on a developer's toes.